CMakeLists.txt

```diff
@@ -29,10 +29,14 @@ add_executable(${PROJECT_NAME} WIN32
 
 set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -DDEBUG")
 
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /guard:cf /Qspectre")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /guard:cf")
+if (NOT ASAN)
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /Qspectre")
+endif ()
+
 set(CMAKE_EXE_LINKER_FLAGS  "${CMAKE_EXE_LINKER_FLAGS} /guard:cf /DYNAMICBASE")
 
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /wd4996 /wd4267 /wd4244")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /wd4996 /wd4267 /wd4244 /wd5072")
```

Comment:
I really think we should document what these numbers mean here. Sure, I can google it. But a comment would be better.

Reply:
I changed the way how ASAN is enabled and this is not needed anymore.